rest_framework/schemas/openapi.py

```diff
@@ -442,22 +443,27 @@ def _map_field(self, field):
                 content['format'] = field.protocol
             return content
 
-        # DecimalField has multipleOf based on decimal_places
         if isinstance(field, serializers.DecimalField):
-            content = {
-                'type': 'number'
-            }
-            if field.decimal_places:
-                content['multipleOf'] = float('.' + (field.decimal_places - 1) * '0' + '1')
-            if field.max_whole_digits:
-                content['maximum'] = int(field.max_whole_digits * '9') + 1
-                content['minimum'] = -content['maximum']
-            self._map_min_max(field, content)
-            return content
+            if field.coerce_to_string:
```

Comment:
I think getattr is necessary since `coerce_to_string` isn't always initialized on the field
```suggestion
            if getattr(field, 'coerce_to_string', api_settings.COERCE_DECIMAL_TO_STRING):
```

Reply:
Updated.